config/routes.rb

```diff
@@ -5,11 +5,11 @@
 
   root "homes#index"
 
-  resources :coffeeshops, only: [:index, :show]
+  resources :coffeeshops, only: [:index, :show, :create, :new]
```

Comment:
We want to create a separation of regular routes vs API controller actions to avoid duplicates. It looks like the creation of coffeeshops happens in react with an API controller, so we don't need to define these "regular" controller routes. If the index and show are handled by ERB templates, then they shouldn't be listed below as API endpoints

Reply:
Thank you for this info, it provided a lot of solid clarifications to better understand what is going on under the hood